test/body.jl

```diff
@@ -1,53 +1,58 @@
 using HTTP.Messages
 
-@testset "HTTP.Bodies" begin
-
-    @test String(take!(Body("Hello!"))) == "Hello!"
-    @test String(take!(Body(IOBuffer("Hello!")))) == "Hello!"
-    @test String(take!(Body(Vector{UInt8}("Hello!")))) == "Hello!"
-    @test String(take!(Body())) == ""
-
-    io = Base.BufferStream()
-    @async begin
-        write(io, "Hello")
-        sleep(0.1)
-        write(io, "!")
-        sleep(0.1)
+@test_skip @testset "HTTP.Bodies" begin
+    @testset "Body" begin
+        @test String(take!(Body("Hello!"))) == "Hello!"
```

Comment:
I think `Body` can be changed to `HTTP.bodybytes`; though these tests are pretty old and crufty, so I'm not sure if they're still useful (i.e. we might just remove them). But if you change to `HTTP.bodybytes`, and they work, we can keep them.

Reply:
Looking more into it, `bodybytes` are being tested in `test/messages.jl`. This file can be removed.